packages/compass-import-export/src/modules/export.js

```diff
@@ -426,70 +447,23 @@ export const startExport = () => {
 
     let exportSucceded = true;
 
-    const spec = exportData.isFullCollection
-      ? { filter: {} }
-      : exportData.query;
-    const numDocsToExport = exportData.isFullCollection
-      ? await fetchDocumentCount(dataService, ns, spec)
-      : exportData.count;
-    // filter out only the fields we want to include in our export data
-    const projection = Object.fromEntries(
-      Object.entries(exportData.fields).filter(
-        (keyAndValue) => keyAndValue[1] === 1
-      )
-    );
-    if (
-      Object.keys(projection).length > 0 &&
-      (undefined === exportData.fields._id || exportData.fields._id === 0)
-    ) {
-      projection._id = 0;
-    }
-    log.info(
-      mongoLogId(1001000083),
-      'Export',
-      'Start reading from collection',
-      {
-        ns,
-        numDocsToExport,
-        spec,
-        projection,
-      }
-    );
-    const progress = createProgressStream({
-      objectMode: true,
-      length: numDocsToExport,
-      time: 250 /* ms */,
-    });
-
-    progress.on('progress', function (info) {
-      dispatch(onProgress(info.percentage, info.transferred));
-    });
-
-    log.info(mongoLogId(1001000084), 'Export', 'Start writing to file', {
+    const dest = fs.createWriteStream(exportData.fileName);
```

Comment:
We used to create this write stream and the export source in the `try` - is there a reason we want to move it out? Is there any way it could fail that we want to catch? Maybe nothing to change.